x/gov/test_common.go

```diff
@@ -155,6 +156,30 @@ func testProposal() Content {
 	return NewTextProposal("Test", "description")
 }
 
+type contextKey int
+
+const contextKeyBadProposalKey contextKey = iota
+
+// badProposalHandler implements a governance proposal handler that is identical
+// to the actual handler except this fails if some given value in the context is
+// odd.
```

Comment:
I think it would be more straightforward to have "some given value" be a bool instead of an integer. Fails when false, passes when true. A bit clearer so I don't have to think about my nemesis the `%` operand

Reply:
lmao  🆗